src/voronoi.rs

```diff
@@ -46,45 +51,49 @@ mod test {
         let mut voronoi = input.voronoi(None::<&GGeometry>, 0., false).unwrap();
 
         let expected_output = "GEOMETRYCOLLECTION (
-            POLYGON ((25 38, 25 295, 221.20588235294116 210.91176470588235, 170.024 38, 25 38)), 
-            POLYGON ((400 369.6542056074766, 400 38, 170.024 38, 221.20588235294116 210.91176470588235, 400 369.6542056074766)), 
+            POLYGON ((25 38, 25 295, 221.20588235294116 210.91176470588235, 170.024 38, 25 38)),
+            POLYGON ((400 369.6542056074766, 400 38, 170.024 38, 221.20588235294116 210.91176470588235, 400 369.6542056074766)),
             POLYGON ((25 295, 25 395, 400 395, 400 369.6542056074766, 221.20588235294116 210.91176470588235, 25 295)))";
 
         let mut expected_output = GGeometry::new_from_wkt(expected_output).unwrap();
 
         expected_output.normalize().unwrap();
         voronoi.normalize().unwrap();
 
-        let same = expected_output.equals(&voronoi).unwrap();
-        assert!(same);
+        assert_eq!(
```

Comment:
Here and next test; due to floating point precision issues there is a tiny amount of difference beyond 10 decimal places.  The fix was to change from equality test to comparing WKT at suitable precision.